andela/krypton-ah-frontend
# #162414128  Implement actions for liking and disliking an article

**What does this PR do?**
Impletement the like and dislike article sessions

Description of Task to be completed?
- Send the required details to the database
- Dispatch appropriate actions

How should this be manually tested?
Git clone the repo and cd into it
npm run dev to start the server
open the redux dev tools to check the application state

What are the relevant pivotal tracker stories?
[link](#162414128 )
